extension/intents/nicknames/nicknames.toml

```diff
@@ -60,4 +60,22 @@ match = """
 phrase = "delete the page called news"
 
 [[nicknames.removePageName.example]]
-phrase = "remove this tab name"
\ No newline at end of file
+phrase = "remove this tab name"
+
+[nicknames.pause]
+description = "Pauses a routine"
+match = """
+  pause routine
+"""
+
+[[nicknames.pause.example]]
+phrase = "pause routine"
```

Comment:
There doesn't need to be an example here. If we describe it somewhere, it should be on the routine page itself.